README.md

```diff
@@ -52,18 +52,36 @@
 
 ## Contributing
 
-## License
+Arrow-meta project is split into 4 packages:
+- compiler-plugin: Kotlin compiler plugin
+- gradle-plugin: Plugin for Gradle for activating Kotlin compiler plugin
+- idea-plugin: Plugin for Intellij IDEA for IDE integration (icons, autocomplete etc.)
+- testing-plugin: Tools for testing compiler and idea plugin
 
-## Credits
+Quick start development environment:
+
+- Run `publishAndRunIde` Gradle task
+- Import sample project (https://github.com/jereksel/arrow-meta-example) in new IDE instance
```

Comment:
Feel free to fork it

Reply:
https://github.com/arrow-kt/arrow-meta/pull/2

Reply:
@jereksel you have been invited :) feel free to port the sample project over to a module. Welcome to Meta!